app/views/sessions/unauthenticated.haml

```diff
@@ -0,0 +1,5 @@
+%h2 Unauthenticated
+%p Sorry! You don't appear to be a user in our system.
+%p If you are an ST dispatcher and believe you should
+be authorized, please contact your administrator.
+%br
```

Comment:
I'd suggest round-three's equivalent for a good template here. Also, I'd suggest not ending a page with a `br` tag.